spec/spec_helper.rb

```diff
@@ -189,7 +189,7 @@ def setup_runtime_log
   config.raise_errors_for_deprecations!
 
   # sometimes stuff hangs -> do not hang everything
-  config.include(Module.new {def test_timeout;30;end })
+  config.include(Module.new {def test_timeout;300;end })
```

Comment:
why do all tests need 5 minutes now ... can we get this back to everything gets 30s and the rails ones get 5 minutes ?

Reply:
These tests were also sometimes timing out prior to this PR, and they are also sometimes passing without the modification in the timeout. I'm just increasing the timeout so that they pass more reliably. The new value, I took it from the unused let being removed in https://github.com/grosser/parallel_tests/pull/663/commits/326d08fa5602796fd2d86033f8158e2c00ed6b6d. Instead, I'll dig into why that let got unused and try to restore it.

Reply:
Ah ok, I misunderstood this. I see what you're doing now. Let me update it.

Reply:
I'm unsure which value to use, though, I guess... 400? Or I can leave it as it is and just restart builds the few times they timeout?

I proposed to just remove the timeout because from all the times I have run the specs, I haven't seen a single hang, just some slow specs for which the timeout is not appropriate.